### PR TITLE
Add stddev and stdvar aggregation functions.

### PR DESCRIPTION
**Pull request by brian-brazil**
This adds the population standard deviation and
variance as aggregation functions, useful for
spotting how many standard deviations some samples
are from the mean.

### CODE REVIEW

File: rules/ast/ast.go [hunk 1]
```diff
@@ -509,6 +518,12 @@ func (node *VectorAggregation) Eval(timestamp clientmodel.Timestamp) Vector {
 				}
 			case Count:
 				groupedResult.groupCount++
+			case Stdvar:
+				fallthrough
```
Comment (juliusv) [inline rules/ast/ast.go]: Could this just be:

`case Stdvar, Stddev:`

Comment (brian-brazil) [inline rules/ast/ast.go]: Yip, thanks.

**Comment by juliusv** [comment]
:+1: 

/cc @fabxc for forward-porting to new promql implementation

**Comment by grobie** [comment]
Oh nice!

**Pull request merged**